xbmc/cores/AudioEngine/Engines/ActiveAE/AudioDSPAddons/ActiveAEDSP.h

```diff
@@ -399,13 +395,14 @@ namespace ActiveAE
     static const int        m_StreamTypeNameTable[];                    /*!< Table for stream type strings related to type id */
     bool                    m_isActive;                                 /*!< set to true if all available dsp addons are loaded */
     AE_DSP_ADDONMAP         m_addonMap;                                 /*!< a map of all known audio dsp addons */
+    std::list<AE_DSP_ADDON> m_addonToDestroy;                           /*!< a map of all known audio dsp addons */
```

Comment:
If it's for speed a vector may be a lot better, better cache locality and prefetching when reading from it. Either way works, just pointing it out